server/src/db/migrations/20240501141059_createReviews.cjs

```diff
@@ -0,0 +1,23 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+  return knex.schema.createTable("reviews", (table) => {
+      table.bigIncrements("id")
+      table.string("reviewText").notNullable()
```

Comment:
this is redundant. Replace this with just `text` or `body` or `content`. The name of the table should ideally not be in the name